src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function (fog) {
```

Comment:
Admittedly, I wasn't super thrilled about that change either. However, I did it this way to keep in line with how the rest of ObjectLoader works. If you look in ObjectLoader you'll see arguments to other objects are passed in a similar fashion.